packages/utils/src/types/channelEntry.ts

```diff
@@ -126,6 +126,21 @@ export class ChannelEntry {
   }
 
   public getId() {
-    return generateChannelId(this.source, this.destination)
+    return generateChannelId(this.source.toAddress(), this.destination.toAddress())
+  }
+
+  public static createMock(): ChannelEntry {
+    const pub = PublicKey.createMock()
+    return new ChannelEntry(
+      pub,
+      pub,
```

Comment:
Why not make second public key different from first one? There should be no "real" on-chain event with `address(source) == address(destination)`

Reply:
Will postpone until we need this in a test.